draft-ietf-httpbis-rand-access-live.xml

```diff
@@ -246,139 +270,148 @@
       </t>
       <t>
-        A Client can indicate support for handling indeterminate-length byte-range responses by providing a Very Large Value for the last-byte-pos in the byte-range request. For example, a Client can perform a byte-range GET request of the form:
+        A client can indicate support for handling indeterminate-length byte-range responses by providing a Very Large Value for the last-byte-pos in the byte-range request. For example, a client can perform a byte-range GET request of the form:
       </t>
 <figure><artwork type="example">
-   Range: bytes=1230000-999999999999
+    GET /resource HTTP/1.1
+    Range: bytes=1230000-999999999999
 </artwork></figure>
       <t>    
         where the last-byte-pos in the Request is much larger than the last-byte-pos returned in response to an open-ended byte-range HEAD request, as described above.
       </t>
-
-    </section>
-      <section anchor="live-range-responses" title="Byte-Range Responses Beyond the Randomly Accessible Byte Range">
       <t>
-        A Server may indicate that it is supplying a continuously aggregating ("live") response by supplying the Client request's last-byte-pos in the Content-Range response header. 
+        In response, a server may indicate that it is supplying a continuously aggregating ("live") response by supplying the client request's last-byte-pos in the Content-Range response header. 
       </t>
       <t>
         For example:
       </t>
 <figure><artwork type="example">
-   Range: bytes=1230000-999999999999
+    GET /resource HTTP/1.1
+    Range: bytes=1230000-999999999999
 </artwork></figure>
       <t>
         could return
       </t>        
 <figure><artwork type="example">
-   Content-Range: bytes 1230000-999999999999/*
+    HTTP/1.1 206 Partial Content
+    Content-Range: bytes 1230000-999999999999/*
 </artwork></figure>
       <t>        
-        from the Server to indicate that the response will start at byte 1230000 and continues indefinitely to include all aggregated content, as it becomes available.
+        from the server to indicate that the response will start at byte 1230000 and continues indefinitely to include all aggregated content, as it becomes available.
       </t>
       <t>
-        A Server that doesn't support or supply a continuously aggregating ("live") response should supply the currently satisfiable byte range, as it would with an open-ended byte request.
+        A server that doesn't support or supply a continuously aggregating ("live") response will supply the currently satisfiable byte range, as it would with an open-ended byte request.
       </t>
       <t>
         For example:
       </t>
 <figure><artwork type="example">
-   Range: bytes=1230000-999999999999
+    GET /resource HTTP/1.1
+    Range: bytes=1230000-999999999999
 </artwork></figure>
       <t>
-        could return
+        will return
```

Comment:
maybe "will return a range covering only those bytes that are currently available:"

Reply:
fixed